site/public/js/pdf/PDFInitToolbar.js

```diff
@@ -57,8 +57,10 @@ function renderPDFToolbar() {
                 case 'eraser':
                     currentTool = 'eraser';
                     PDFAnnotate.UI.enableEraser();
+                    PDFAnnotate.UI.disableEdit();
```

Comment:
I'm still not clear on how we would hit an instance where on hitting `eraser`, `disableEdit` would not have already been called above?

Reply:
I suppose you're right, I just added that line since I somehow managed to get to a state where scrolling with the eraser was leaving tiny marks where I first initiated the scroll. I haven't been able to reproduce it with/without that line so I can remove that line if you want.

Reply:
Please do.